CMakeLists.txt

```diff
@@ -43,6 +43,10 @@ option(WITH_DYNAMIC_CUDA_TOOLKIT "Links CUDA toolkit libraries dynamically (NPP,
 
 option(WITH_DYNAMIC_NVIMGCODEC "Links nvimgcodec library dynamically during runtime" ON)
 
+option(BUILD_WITH_ASAN "Build with ASAN" OFF)
+option(BUILD_WITH_LSAN "Build with LSAN" OFF)
+option(BUILD_WITH_UBSAN "Build with UBSAN" OFF)
+
```

Comment:
Is it related?

Reply:
Kinda, without it the signatures are not generated in a default build